ormconfig.json

```diff
@@ -1,6 +1,6 @@
 {
    "type": "postgres",
-   "host": "localhost",
+   "host": "database",
```

Comment:
this should not be merged into master - host should be environmentalized - work for env vars comes from testing infrastructure

Reply:
@zsimonelli this will be part of your dockerizing ticket